Tests/TestProjectGenerator/Reqnroll.TestProjectGenerator/ProjectBuilder.cs

```diff
@@ -14,11 +14,11 @@ public class ProjectBuilder
         public const string NUnit3PackageName = "NUnit";
         public const string NUnit3PackageVersion = "3.13.1";
         public const string NUnit3TestAdapterPackageName = "NUnit3TestAdapter";
-        public const string NUnit3TestAdapterPackageVersion = "3.17.0";
+        public const string NUnit3TestAdapterPackageVersion = "5.2.0";
```

Comment:
Doubting if we should update this one.

Reply:
Well the tests are green right? So it works with NUnit 3?

Also in the changelog it doesn't say it's only for NUnit 4+

<img width="677" height="327" alt="Image" src="https://github.com/user-attachments/assets/a00c823d-7fcc-4b91-a0e7-0adeb1060649" />

Reply:
@304NotModified thx for checking. yep, we can keep it then